monkey/agent_plugins/credentials_collectors/ssh/src/plugin.py

```diff
@@ -14,12 +13,18 @@
 )
 from common.types import AgentID, Event
 
+logger = logging.getLogger(__name__)
+
+try:
+    import pwd
+except ModuleNotFoundError as err:
+    logger.debug(f"Most likely running on an unsupported OS: {err}")
```

Comment:
Do we need to do this in the plugin? The plugin is not supported on Windows so I don't see how it is an issue.

Reply:
Read the commit message. 
https://github.com/guardicore/monkey/pull/3331/commits/831ad89526bbe2c4094718231384167eb1805aa5

Reply:
We do.
Read the commit message. https://github.com/guardicore/monkey/pull/3331/commits/831ad89526bbe2c4094718231384167eb1805aa5

Reply:
Found a better way. See [41ea0e6](https://github.com/guardicore/monkey/pull/3331/commits/41ea0e62ad80b88d80336a1f92d3ed8937ff2514).

Reply:
Nice job!

Reply:
Can we just @pytest.skip the test and the fixture? Does @pytest.skip work on fixtures?

Reply:
We'd be skipping both tests anyway. What's wrong with skipping the whole file?

Reply:
Skipping the whole file makes sense. My main objection is all of the ignores for the type checker. But it's not really a big deal.

Would this work?
```
pytest.importskip("pwd")
import pwd
```

Reply:
I suppose the type errors will still appear in windows.

We should pull these commits out of this PR and push them to develop.